### ISSUE TITLE
A11y: Mini cart - Shopping cart table doesn't have a sufficient semantic html structure

### ISSUE THREAD

**Issue by tieupmedia** [shopware/platform#5224]
*PHP Version:* 8.2   | *Shopware Version:* 6.6.6.7  | *Affected area / extension:* Platform(Default) 

 ----

*_Expected behaviour:_*

As the element is similar to a list, it would make sense to turn it into a html list. The implementation as a html list is certainly the simplest way in this situation. 

Alternatively, you can also work with headings or possibly even turn it into a table. 

*_Actual behaviour:_*

After adding an article to the shopping cart, the offcanvas mini cart opens. This contains a list of the choosen products. Programmatically (in the source code) this information is not sufficiently semantically structured. It is only a collection of `div`. The correct labeling in the source code is also important, as screen reader users are then offered further navigation options.

---

Violation relating to [EN 301 549](https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf#%5B%7B%22num%22%3A120%2C%22gen%22%3A0%7D%2C%7B%22name%22%3A%22XYZ%22%7D%2C54%2C517%2C0%5D), https://www.w3.org/WAI/WCAG22/quickref/?versions=2.1#info-and-relationships and https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

*_How to reproduce:_*

Go to the default storefront and add a product to the cart, so that the mini cart opens from offcanvas. Inspect the source code.

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-39191**



[created from NEXT-39191, comment 505783]
